static_src/test/unit/actions/service_actions.spec.js

```diff
@@ -6,6 +6,15 @@ import cfApi from '../../../util/cf_api.js';
 import serviceActions from '../../../actions/service_actions.js';
 import { serviceActionTypes } from '../../../constants.js';
 
+function assertAction(spy, type, params) {
+  expect(spy).toHaveBeenCalledOnce();
+  let arg = spy.getCall(0).args[0];
```

Comment:
my only worry is the indices used for `getCall` and `args` i really doubt the order of the call or args will ever matter. but i'm just thinking how this will be used in case some other developer comes along trying to add tests.